ILSpy.ReadyToRun/ILSpy.ReadyToRun.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <AssemblyName>ILSpy.ReadyToRun.Plugin</AssemblyName>
+    <LangVersion>7.2</LangVersion>
+
+    <GenerateAssemblyInfo>False</GenerateAssemblyInfo>
+
+    <EnableDefaultItems>false</EnableDefaultItems>
+
+    <BaseAddress>6488064</BaseAddress>
```

Comment:
This is probably wrong - I copied it from BamlDecompiler - apparently, we do not want duplicated values as that would force relocation. What do we want to do here?

Reply:
Do we even need to specify a base address? I think it won't matter anyway as ASLR discards the value IIRC.